### PR TITLE
Partial fix for Snapdragon HITL mode

### PR DESCRIPTION
**Pull request by jywilson**

### CODE REVIEW

File: src/modules/commander/commander.cpp [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ int commander_thread_main(int argc, char *argv[])
 #endif
 
 	if (argc > 2) {
-		if (!strcmp(argv[3],"-hil")) {
+		if (!strcmp(argv[2],"-hil")) {
```
Comment (LorenzMeier) [inline src/modules/commander/commander.cpp]: I need to re-check this on Pixhawk - if you need this to go in fast we can ifdef this to pull it in.

Comment (julianoes) [inline src/modules/commander/commander.cpp]: The removed line looks wrong and would segfault.

Comment (bharath374) [inline src/modules/commander/commander.cpp]: Was it checked on Pixhawk yet? Or should we #ifdef this for QuRT? Yes the removed line did segfault.

**Comment by bharath374** [comment]
Guys, is anything holding up thus pull request? Looks like the issue was introduced here: https://github.com/PX4/Firmware/pull/4277/files Any thoughts?

**Comment by LorenzMeier** [comment]
Rebased and applied.

**Pull request closed without merging**